src/app/mail/mail-detail/mail-detail.component.ts

```diff
@@ -1,18 +1,18 @@
-import { Component, OnInit } from '@angular/core';
-import { Mail } from '../../store/models/mail.model';
-import { ActivatedRoute } from '@angular/router';
-import { Store } from '@ngrx/store';
-import { ClearMailDetail, GetMailDetail, ReadMail } from '../../store/actions/mail.actions';
-import { Observable } from 'rxjs/Observable';
-import { AppState, MailState } from '../../store/datatypes';
-import { OnDestroy, TakeUntilDestroy } from 'ngx-take-until-destroy';
-import { OpenPgpService } from '../../store/services';
+import { Component, OnInit } from "@angular/core";
```

Comment:
Please don't change the single quotes to double quotes.
I asked you to change your IDE, and use tslint.
Please do it.